### DISCUSSION TITLE
Import suggestions

### DISCUSSION THREAD

**Discussion by maxim092001** [scalameta/nvim-metals#331] [Q&A] [answered]
Hi there!

I was using IntelliJ for a long time and decided to get into nvim and metals, thanks for this [guide](https://www.youtube.com/watch?v=ihKdrhPaZjg)!

However, I didn't find any option to get suggestions for imports. Example in screenshots comparing with IntelliJ. Is there any option to do the same thing in metals?

<img width="416" alt="image" src="https://user-images.githubusercontent.com/35135337/151691683-637aaade-1153-4287-a60f-2334ef54a8fc.png">

<img width="439" alt="image" src="https://user-images.githubusercontent.com/35135337/151691677-967f1cf2-2e21-4f31-9ed6-c2172a0b0a18.png">

**Comment by antosha417** [comment, marked as answer]
I use https://github.com/nvim-telescope/telescope.nvim for this. With this command:

<cmd>lua require"telescope.builtin".lsp_code_actions()<cr>

**Reply by landerlo** [reply]
Alternatively, just map the nvim provided  ```vim.lsp.buf.code_action()```

**Reply by maxim092001** [reply]
Thanks! ```vim.lsp.buf.code_action()``` was actually in example [configuration](https://github.com/scalameta/nvim-metals/discussions/39), I missed it somehow. :)
